packages/asc-ui/src/components/DescriptionList/DescriptionListStyle.ts

```diff
@@ -0,0 +1,38 @@
+import styled from 'styled-components'
+import { breakpoint, themeColor, themeSpacing } from '../../utils'
+
+export const DescriptionListWrapper = styled.dl`
+  width: 100%;
+  margin-top: 0;
+`
+
+export const DescriptionListItemWrapper = styled.div`
+  display: flex;
+  padding: 10.5px 0;
```

Comment:
Why this specific value again?

Reply:
design wants the items to have a height of 40px